lib/node_modules/@stdlib/lapack/base/dlae2/README.md

```diff
@@ -0,0 +1,221 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlae2
+
+> Compute the eigenvalues of a 2x2 symmetric matrix.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dlae2 = require( '@stdlib/lapack/base/dlae2' );
+```
+
+#### dlae2( A, B, C, Out )
+
+Computes the eigenvalues of a 2x2 symmetric matrix.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var out = new Float64Array( 2 );
+out = dlae2( 2.0, 3.0, 4.0, out );
+// out => <Float64Array>[ 6.16227766016838, -0.1622776601683793 ]
+```
+
+The function has the following parameters:
+
+-   **A**: the (0,0) element of matrix.
+-   **B**: the (0,1) & (1,0) elements of matrix.
+-   **C**: the (1,1) element of matrix.
+-   **Out**: output [`Float64Array`][mdn-float64array] containing eigenvalue of larger and smaller absolute value respectively.
+
+Note that indexing is relative to the first index. To introduce an offset, use [`typed array`][mdn-typed-array] views.
+
+<!-- eslint-disable stdlib/capitalized-comments -->
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+// Initial arrays...
+var OUT0 = new Float64Array( [ 0.0, 0.0, 0.0 ] );
+
+// Create offset views...
+var OUT1 = new Float64Array( OUT0.buffer, OUT0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+
+dlae2( 2.0, 3.0, 4.0, OUT1 );
+// OUT0 => <Float64Array>[ 0.0, 6.16227766016838, -0.1622776601683793 ]
+```
+
+#### dlae2.ndarray( A, B, C, Out, strideOut, offsetOut )
+
+Computes the eigenvalues of a 2x2 symmetric matrix using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var out = new Float64Array( 2 );
+out = dlae2.ndarray( 2.0, 3.0, 4.0, out, 1, 0 );
+// out => <Float64Array>[ 6.16227766016838, -0.1622776601683793 ]
```

Comment:
Use approximate values, rather than exact, lest the implementation change and thus the precision of the returned results.